cirq/google/calibration/phased_fsim_test.py

```diff
@@ -104,6 +110,48 @@ def test_floquet_to_calibration_layer():
     )
 
 
+def test_xeb_to_calibration_layer():
+    q_00, q_01, q_02, q_03 = [cirq.GridQubit(0, index) for index in range(4)]
+    gate = cirq.FSimGate(theta=np.pi / 4, phi=0.0)
+    request = XEBPhasedFSimCalibrationRequest(
+        gate=gate,
+        pairs=((q_00, q_01), (q_02, q_03)),
+        options=XEBPhasedFSimCalibrationOptions(
+            n_library_circuits=22,
+            fsim_options=XEBPhasedFSimCharacterizationOptions(
+                characterize_theta=True,
+                characterize_zeta=True,
+                characterize_chi=False,
+                characterize_gamma=False,
+                characterize_phi=True,
+            ),
+        ),
+    )
+    assert request.to_calibration_layer() == cirq.google.CalibrationLayer(
+        calibration_type='xeb_phased_fsim_characterization',
+        program=cirq.Circuit([gate.on(q_00, q_01), gate.on(q_02, q_03)]),
+        args={
+            'n_library_circuits': 22,
+            'n_combinations': 10,
+            'cycle_min': 3,
+            'cycle_max': 100,
+            'cycle_step': 20,
+            'fatol': 5e-3,
+            'xatol': 5e-3,
+            'characterize_theta': True,
+            'characterize_zeta': True,
+            'characterize_chi': False,
+            'characterize_gamma': False,
+            'characterize_phi': True,
+            'theta_default': 0,
```

Comment:
Good question. This would cause issues. However, the actual result of `request.to_calibration_layer()` uses floats (in fact, that change is part of this PR. See the changes on `cirq.experiments.xeb_fitting:166`).

For this particular test, the `request.to_calibration_layer()` request (which uses float values) still reports equality with the desired `CalibrationLayer` constructed here (which uses ints). This is because in Python `0.0 == 0` is `True`. 

I've changed these values in the test object to floats and added a test that the layer can be serialized to a proto. Of course, we must be careful as everything gets turned into a float, which is relevant for deserialization.